torch/csrc/distributed/c10d/UCCForNCCL.hpp

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include <string>
+#include <vector>
+#include <cassert>
+#include <memory>
+
+#include <ATen/DynamicLibrary.h>
+
+namespace c10d {
+
+std::shared_ptr<at::DynamicLibrary> loadTorchUCC() {
```

Comment:
Let's make this interface as inline? Otherwise it causes conflicting symbol in building time. Thanks!

Reply:
done

Reply:
Thanks! Imported and doing another round of tests